app/Nova/User.php

```diff
@@ -809,4 +810,25 @@ private static function adminCanRun(NovaRequest $request): bool
     {
         return $request->user()->hasRole('admin');
     }
+
+    /**
+     * Build a Scout search query for the given resource.
+     *
+     * @param  \Laravel\Scout\Builder  $query
+     */
+    public static function scoutQuery(NovaRequest $request, $query): \Laravel\Scout\Builder
+    {
+        if ($request->field === 'projectManager' || $request->field === 'primaryContact') {
```

Comment:
This is going to be a little weird in test because the Meilisearch index is out of date. Lmk if I need to import things.